plugins/webgui/public/translate/en-US.js

```diff
@@ -88,23 +88,23 @@ module.exports = {
   '账号设置': 'Account Configuration',
 
   '用户注册失败': 'Registration failed',
-  '网络异常，请稍后再试': 'Network abnormal, please try it later',
-  '用户名或密码错误': 'User name or password is incorrect',
-  '该用户尚未注册的': 'This user has not been registered',
-  '请输入正确的用户名格式': 'The right format of user name is required',
-  '密码重试次数已达上限\n请稍后再试': 'The limit of invalid password attempts has been reached\n Please try it later',
-  '验证码发送错误': 'Code sending error',
-  '发送错误，请更换邮箱尝试': 'Code sending error: change your email and try it again',
-  '请求过于频繁，请稍后再试': 'Too frequent requests, please try it later',
-  '当前时段尚未开放注册': 'The registration is closed currently',
-  '重置密码链接已发至您的邮箱，\n请注意查收': 'A password reset link has been sent to your email,\n please check it ',
-  '重置密码链接已经发送，\n请勿重复发送': 'The password reset link has been sent, \n please do not request it again',
-  '请输入正确的邮箱地址': 'A right format of email is required',
+  '网络异常，请稍后再试': 'Network failure. Please try again later.',
+  '用户名或密码错误': 'Incorrect username or password.',
+  '该用户尚未注册的': 'No such user',
+  '请输入正确的用户名格式': 'Username invalid.',
+  '密码重试次数已达上限\n请稍后再试': 'Too many login attempts.\n Please try again later.',
+  '验证码发送错误': 'Unable to send verification code.',
+  '发送错误，请更换邮箱尝试': 'Unable to send verification code. Try to use another E-mail address.',
```

Comment:
Don't forget to use 'please'. Again, 'email address'. 'Please try a different email address'